packages/synapse-react-client/src/components/CookiesNotification/CookiePreferencesDialog.tsx

```diff
@@ -0,0 +1,153 @@
+            // But we need to specify the Checkbox onChange handler, otherwise the Checkbox will toggle the value back!
+            onChange={() => {
+              props.onChange(props.checked)
+            }}
+            checked={props.checked}
+            label=""
+          />
+        )}
+      </Box>
+      <Typography variant={'body1'} sx={{ fontWeight: 400, color: 'grey.700' }}>
+        {description}
+      </Typography>
+    </Box>
+  )
+}
+
+export default function CookiePreferencesDialog(
+  props: CookiePreferencesDialogProps,
+) {
+  const [cookiePreferences, setCookiePreferences] = useCookiePreferences()
+  const { onSave, onHide, isOpen } = props
+  const [isFunctionalCookiesAllowed, setIsFunctionalCookiesAllowed] =
+    useState<boolean>(cookiePreferences.functionalAllowed)
+  const [isAnalyticsCookiesAllowed, setIsAnalyticsCookiesAllowed] =
+    useState<boolean>(cookiePreferences.analyticsAllowed)
+
+  return (
+    <Dialog maxWidth="md" open={isOpen} onClose={onHide} sx={{ zIndex: 1500 }}>
+      <DialogBaseTitle title="Privacy Preferences" onCancel={onHide} />
+      <DialogContent sx={{ paddingRight: '20px' }}>
+        <Typography variant={'body1'}>
+          Sage Bionetworks uses cookies and other browser storage methods to
+          save information about how you use our websites, and to make our
+          websites work the way you expect them to.
+        </Typography>
+        <Typography variant={'body1'} sx={{ marginBottom: '30px' }}>
+          Read our{' '}
+          <Link href="https://s3.amazonaws.com/static.synapse.org/governance/SynapsePrivacyPolicy.pdf">
+            privacy policy
+          </Link>
+          .
+        </Typography>
+        <CookiePreferenceItem
+          title="Strictly necessary cookies"
+          description="Information about your authentication, and other critical data. For example, after you log in successfully, we save a token in your browser that tells our website what level of access you are allowed to have. You can’t turn off these cookies, because the site wouldn’t function without them."
+          constantValue="Always Active"
+        />
+        <CookiePreferenceItem
+          title="Functional cookies"
+          description="This includes important customization preferences, for example, your local time, and whether you are using experimental features. You can turn off this setting, however it may impact your experience of using our website."
+          checked={isFunctionalCookiesAllowed}
+          onChange={setIsFunctionalCookiesAllowed}
+        />
+        <CookiePreferenceItem
+          title="Analytics cookie"
+          description="If you allow us to, we store information about how you use our website. This helps us identify problems and improve our products. While we do not attempt to track you, it may be possible to identify individual users. Deactivating this setting will not affect your experience of using our websites."
```

Comment:
Plural agreement (or go the otherway so it's all singular "website")

```suggestion
        <CookiePreferenceItem
          title="Functional cookies"
          description="This includes important customization preferences, for example, your local time, and whether you are using experimental features. You can turn off this setting, however it may impact your experience of using our websites."
          checked={isFunctionalCookiesAllowed}
          onChange={setIsFunctionalCookiesAllowed}
        />
        <CookiePreferenceItem
          title="Analytics cookie"
          description="If you allow us to, we store information about how you use our websites. This helps us identify problems and improve our products. While we do not attempt to track you, it may be possible to identify individual users. Deactivating this setting will not affect your experience of using our websites."
```